admin_guide/garbage_collection.adoc

```diff
@@ -86,18 +86,18 @@ Each spin of the garbage collector loop goes through the following steps:
 1. Retrieve a list of available containers.
 2. Filter out all containers that are running or are not alive longer than
 the `*minimum-container-ttl-duration*` parameter.
-3. Determine pod membership of each remaining container.
-4. Remove all unidentified containers (with no pod membership).
-5. For each pod that contains more containers than the
-`*maximum-dead-containers-per-container*` parameter, sort the containers by
+3. Decompose all remaining containers into equivalence classes based on pod and image name membership.
+4. Remove all unidentified containers (containers with unrecognizable name).
```

Comment:
Positive. Here [1], all containers with unparsable name are stored into unidentifiedContainers list. And here [2], they are deleted.

Which is not good as GC can remove key openshift containers if they are not used sufficiently frequently.

[1] https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/dockertools/container_gc.go#L156
[2] https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/dockertools/container_gc.go#L185-L187